### ISSUE TITLE
Emacs/Vim cards

### ISSUE THREAD

**Issue by drkmrin78** [jstpcs/lnxpcs#56]
Any chance you can make either?

**Comment by jstpcs** [comment]
to my shame, i've never used emacs but vim definitely and it deserves a card. i'd love to create one.  i created one pic for vim already by the way, not exactly for vim but for [wq:](https://github.com/jstpcs/lnxpcs/blob/21c8f5ee1f67bb876d9c7b9be1bd0cb86ef9875b/non-distro/vim-shortcut.png) shortcut.

**Comment by drkmrin78** [comment]
I'm an emacs enthusiast. If you like vim try emacs with evil mode; that's the vim integration of emacs.

**Comment by davv-deimos** [comment]
+1 to emacs card

**Comment by jstpcs** [comment]
here are vim cards - [white](https://github.com/jstpcs/lnxpcs/blob/5a208b29ea2851f8881a44317bbf3ee8ecc38031/cards/classic/vim-card.png) card and [dark](https://github.com/jstpcs/lnxpcs/blob/5a208b29ea2851f8881a44317bbf3ee8ecc38031/cards/black/vim-card-black.png) card.

**Comment by davv-deimos** [comment]
They are really cool, I prefer the white version on this one.

So all that is missing now is an Emacs one :P

**Comment by kaushalmodi** [comment]
It would be really cool if something (Org mode,Magit, Evil, .. ?) inspires you to try out Emacs, and then we have cards for Emacs! :)

**Comment by davv-deimos** [comment]
yeah, don't draw without inspiration!!! :D Emacs deserves your best work.

**Comment by Grimscythe1983** [comment]
Might do an Evil Emacs, which is Emacs empowered with Vim bindings. It is essentially Vim 3000 ;)

**Comment by drkmrin78** [comment]
^^^ yes plz, I love evil mode

**Comment by xxzozaxx** [comment]
+1 Emacs
+1 Org-mode ( it's logo will inspire you )
+1 Evil-mode
